packager/mpd/base/period.cc

```diff
@@ -240,11 +239,12 @@ const AdaptationSet* Period::FindOriginalAdaptationSetForTrickPlay(
   MediaInfo media_info_no_trickplay = media_info;
   media_info_no_trickplay.mutable_video_info()->clear_playback_rate();
 
-  std::string key = GetAdaptationSetKey(media_info_no_trickplay);
+  std::string key = GetAdaptationSetKey(media_info_no_trickplay, false);
```

Comment:
s/false/mpd_options_.mpd_params.allow_codec_switching/

It has to be the same as the key we use when setting up |adaptation_set_list_map_|, otherwise |adaptation_sets| below would be empty.